vtex/ab-tester
# Feature/evaluate revenue test

#### What is the purpose of this pull request?
Create new functions to correctly evaluate an AB test that uses revenue as main metric.

#### What problem is this solving?
Revenue AB tests were evaluated as conversion tests.

#### How should this be manually tested?
Manually start a revenue test and see that the stats make sense and everything works.

#### Types of changes
